### PR TITLE
fix: dont render markdown in toc

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[![PR App][icn]][demo] | Ref RM-10928
:-------------------:|:----------:

## 🧰 Changes

Prevents rendering markdown in the ToC

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

**Review by trishaprile — APPROVED** [review]
looks good besides a type error

File: __tests__/plugins/toc.test.tsx [hunk 1]
```diff
@@ -46,4 +46,16 @@ describe('toc transformer', () => {
...
+    const md = `
+# [Title](http://example.com)
+`;
+    const { Toc } = await run(compile(md));
+
+    render(<Toc />);
+
+    expect(screen.findByText('Title')).toBeDefined();
+    expect(screen.queryByText('[', { exact: false })).toBeNull();
+  });
```
Comment (rafegoldberg) [inline __tests__/plugins/toc.test.tsx:60]: this is a very good/legible test!

**Review by rafegoldberg — APPROVED** [review]
I like this PR!

File: __tests__/plugins/toc.test.tsx (hunk 1 above)
Comment (kellyjosephprice) [reply __tests__/plugins/toc.test.tsx:60]: That last expect bugs me, but it was rendering a `[` text node. :shrug:

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v7.6.3</b>](/readmeio/markdown/releases/tag/v7.6.3)
<br>

**Pull request merged**